tendrl/common/utils/command.py

```diff
@@ -5,9 +5,25 @@
 ANSIBLE_MODULE_PATH = "core/commands/command.py"
 LOG = logging.getLogger(__name__)
 
+SAFE_COMMAND_LIST = [
```

Comment:
My opinion is having it in code is better, keeping the security in mind. 
@nthomas-redhat @r0h4n @brainfunked  Please provide your thoughts